lib/generator.js

```diff
@@ -288,6 +288,15 @@ class Generator {
    * @return {Promise}
    */
   generateDirectoryStructure(asyncapiDocument) {
+    const fileNamesForSeperation = {
+      channel: asyncapiDocument.channels(),
+      message: asyncapiDocument.components().messages(),
+      securityScheme: asyncapiDocument.components().securitySchemes(),
+      scheme: asyncapiDocument.components().schemas(),
```

Comment:
Let's use `schema` instead of `scheme`. We're already using `scheme` for "security schemes" and it can be confusing. Regarding usefulness, I actually find it useful for generating type definitions for typed languages.

Reply:
Yep agreed.